apps/video/js/video.js

```diff
@@ -1029,11 +1041,11 @@ function showPlayer(video, autoPlay, enterFullscreen, keepControls) {
       switchLayout(LAYOUT_MODE.fullscreenPlayer);
     }
 
-    dom.durationText.textContent = MediaUtils.formatDuration(
-      dom.player.duration);
+    var formattedTime = MediaUtils.formatDuration(dom.player.duration);
+    dom.durationText.textContent = formattedTime;
```

Comment:
IMO, the code would be more readable if ''formattedTime' was 'formatedDuration', since the currentTime is also formatted below; 'formattedTime' is a bit vague...